docs/docs/upgrade-gatsby-and-dependencies.md

```diff
@@ -0,0 +1,142 @@
+  "gatsby": "^2.15.13",
+}
+```
+
+For major updates follow up with the corresponding guide from the [Release and Migrations](https://www.gatsbyjs.org/docs/releases-and-migration/) reference guide overview.
+
+In case that you are updating Gatsby, be sure to also update Gatsby related plugins, you can identify them because their name starts with `gatsby-`, this only applies to plugins managed in the gatsbyjs/gatsby repo, for community plugins check beforehand if there is a new version you can upgrade to.
+
+## Updating all your dependencies at once
+
+After adding the corresponding annotations into your `package.json` file, you can run the update command in npm or the upgrade command if you are using yarn, remember that you only have to use one of yarn or npm:
+
+```shell
+npm update
+```
+
+```shell
+yarn upgrade
+```
+
+This will upgrade all your packages to the latest [wanted](https://docs.npmjs.com/cli/outdated) version, this means the latest patch in case that you specified that only patch updates were allowed, and so on for minor and major update annotations.
+
+## Upgrade individual dependencies
+
+You can also update one package at the time with the install command in npm, or the add command in yarn, alongside the version that you want to install:
+
+```shell
+npm install <package-name>@<version> --save
+```
+
+```shell
+yarn add <package-name>@<version>
+```
+
+You can specify the version you want to install or upgrade to, in the following formats:
+
+- An specific version after the @
+- An annotated version with \*,^,~ to indicate that you want the latest major, minor or patch release respectively.
+- Use and x instead of a number to indicate that you want the latest mayor (x), minor (&lt;major&gt;.x) or patch release (&lt;major&gt;.&lt;minor&gt;.x).
+
+Remember to follow up with the corresponding guide from the [Release and Migrations](https://www.gatsbyjs.org/docs/releases-and-migration/) reference guide overview, for major upgrades.
+
+## Upgrade Interactively
+
+If you are using yarn you can also use the upgrade-interactive command to manually select which dependencies you want to update:
+
+```shell
+yarn upgrade-interactive
+```
+
+The same behavior can be accomplished in npm through the [npm-check](https://www.npmjs.com/package/npm-check) module. To do that, start by installing the module:
+
+```shell
+npm install npm-check --save-dev
+```
+
+Then add the corresponding script to your package.json file:
+
+```title=package.json
+{
+  "scripts": {
+    "upgrade-interactive": "npm-check --update"
+  }
+}
+```
+
+And finally run the recently added command:
+
+```shell
+npm run upgrade-interactive
+```
+
+## Troubleshooting
+
+Aside from some [specific cases](https://www.gatsbyjs.org/blog/2019-06-18-dropping-support-for-node-6/), upgrading for minor or patch releases should not require you to make changes to your code, but it is recommended to run your suite of tests in case you have one after upgrading Gatsby or its dependencies.
```

Comment:
```suggestion
Aside from some specific cases, such as [Gatsby's dropping of support for Node 6](/blog/2019-06-18-dropping-support-for-node-6/), upgrading for minor or patch releases should not require you to make changes to your code. It is recommended to run your suite of tests (in case you have one) after upgrading Gatsby or its dependencies.
```